src/components/ProductDiv.tsx

```diff
@@ -1,37 +1,31 @@
 import React from "react";
 
 interface ProductDivProps {
-  placingLeft: number;
-  placingTop: number;
+  // placingLeft: number;
+  // placingTop: number;
   imageSrc: string;
   productName: string;
   productPrice: string;
 }
 
 const ProductDiv: React.FC<ProductDivProps> = ({
-  placingLeft,
-  placingTop,
+  // placingLeft,
+  // placingTop,
   imageSrc,
   productName,
   productPrice,
 }) => {
   return (
-    <div
-      className="w-[300px] h-[420px] relative bg-blue-200 rounded-lg" // Change to relative if you don't need flex
-      style={{
-        position: "absolute",
-        left: `${placingLeft}px`,
-        top: `${placingTop}px`,
-      }}
-    >
-      <img
-        src={imageSrc}
-        alt="Product"
-        className="w-[180px] h-[300px] object-cover mb-8 items-center"
-      />
-      {/* Add text-left to ensure left alignment */}
-      <p className="mt-4 text-lg ml-4">{productName}</p>
-      <p className="mt-4 text-lg ml-4">{productPrice}</p>
+    <div className="transition-transform duration-700 ease-in-out hover:scale-105">
+      <div className="w-[250px] h-[250px]  mx-auto mt-8">
```

Comment:
Margins might be problematic (especially since this is not text content), just use padding here instead.

[Why Margin Bad](https://dev.to/moruno21/dont-use-margin-in-css-33f9)